lib/scope.js

```diff
@@ -89,8 +89,12 @@ SymbolDef.prototype = {
 
             // https://github.com/mishoo/UglifyJS2/issues/1095
             // prevent mangle name conflict while using `let` or `const`
-            if (this.orig[0] instanceof AST_SymbolBlockDeclaration)
-                s = s.parent_scope;
+            if (this.orig[0] instanceof AST_SymbolBlockDeclaration) {
```

Comment:
I wonder how adding stuff would improve performance if it doesn't replace slow code with faster code paths.